rpc/build.rs

```diff
@@ -26,6 +26,7 @@ fn main() {
 
     tonic_build::configure()
         .build_server(true)
+        .extern_path(".google.protobuf.Timestamp", "::prost_wkt_types::Timestamp")
```

Comment:
We don't need this, we don't set serde traits here?

Reply:
I'm not sure about serde applicability, but this is added here so that protobuf timestamp maps correctly to `prost_wkt_types::Timestamp`. Without this, the autogen code maps that to `prost_types::Timestamp` which doesn't have `From` trait implementation for conversion.

Reply:
I was under the impression that we don't need to use `prost_wkt_types::Timestamp` as its main drive was to get the serde traits. We can simply use the default protobuf Timestamp.